### PR TITLE
Update PopoutForIncomingPstnCalls and PopoutAppPathForIncomingPstnCalls descriptions

### PR DESCRIPTION
**Pull request by noramicrosoft**
Updating the descriptions for the `PopoutForIncomingPstnCalls` and `PopoutAppPathForIncomingPstnCalls` parameters.

### CODE REVIEW

**Comment by noramicrosoft** [comment]
@microsoft-github-policy-service agree [company="Microsoft"]

**Comment by noramicrosoft** [comment]
@microsoft-github-policy-service agree company="Microsoft"

**Comment by get-itips** [comment]
I see the PR is still draft, is it ready for review? @noramicrosoft @yogkumgit

**Comment by noramicrosoft** [comment]
> I see the PR is still draft, is it ready for review? @noramicrosoft @yogkumgit

Not yet. I'll publish when it's ready.

**Comment by JohanFreelancer9** [comment]
> @JohanFreelancer9

@yogkumgit, I'll edit the PR when it is ready for review.

**Comment by noramicrosoft** [comment]
@get-itips @JohanFreelancer9 it's ready for review now, but not ready to merge

**Review by tseward — APPROVED** [review]

File: skype/skype-ps/skype/New-CsTeamsCallingPolicy.md [hunk 1]
```diff
@@ -343,7 +343,7 @@ Accept wildcard characters: False
 ```
 
 ### -PopoutAppPathForIncomingPstnCalls
-Reserved for internal Microsoft use.
+Setting this parameter allows you to set PopoutForIncomingPstnCalls setting's URL path of the website to launch upon receiving incoming PSTN calls. This parameter accepts an https URL with less than 1024 characters. The URL can contain a `{phone}` placeholder that is replaced with the caller's PSTN number in E.164 format when launched.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsTeamsCallingPolicy.md:346]:
```suggestion
Setting this parameter allows you to set the PopoutForIncomingPstnCalls setting's URL path of the website to launch upon receiving incoming PSTN calls. This parameter accepts an HTTPS URL with less than 1024 characters. The URL can contain a `{phone}` placeholder that is replaced with the caller's PSTN number in E.164 format when launched.

```

File: skype/skype-ps/skype/New-CsTeamsCallingPolicy.md [hunk 2]
```diff
@@ -358,7 +358,7 @@ Accept wildcard characters: False
 ```
 
 ### -PopoutForIncomingPstnCalls
-Reserved for internal Microsoft use.
+Setting this parameter allows you to control the tenant users' ability to launch an external website URL automatically in the browser window upon incoming PSTN calls for specific users/user groups. Valid options are Enabled and Disabled.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsTeamsCallingPolicy.md:361]:
```suggestion
Setting this parameter allows you to control the tenant users' ability to launch an external website URL automatically in the browser window upon incoming PSTN calls for specific users or user groups. Valid options are Enabled and Disabled.

```

File: skype/skype-ps/skype/Set-CsTeamsCallingPolicy.md [hunk 3]
```diff
@@ -345,7 +345,7 @@ Accept wildcard characters: False
 ```
 
 ### -PopoutAppPathForIncomingPstnCalls
-Reserved for internal Microsoft use.
+Setting this parameter allows you to set PopoutForIncomingPstnCalls setting's URL path of the website to launch upon receiving incoming PSTN calls. This parameter accepts an https URL with less than 1024 characters. The URL can contain a `{phone}` placeholder that is replaced with the caller's PSTN number in E.164 format when launched.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTeamsCallingPolicy.md:348]:
```suggestion
Setting this parameter allows you to set the PopoutForIncomingPstnCalls setting's URL path of the website to launch upon receiving incoming PSTN calls. This parameter accepts an HTTPS URL with less than 1024 characters. The URL can contain a `{phone}` placeholder that is replaced with the caller's PSTN number in E.164 format when launched.
```

File: skype/skype-ps/skype/Set-CsTeamsCallingPolicy.md [hunk 4]
```diff
@@ -361,7 +361,7 @@ Accept wildcard characters: False
 ```
 
 ### -PopoutForIncomingPstnCalls
-Reserved for internal Microsoft use.
+Setting this parameter allows you to control the tenant users' ability to launch an external website URL automatically in the browser window upon incoming PSTN calls for specific users/user groups. Valid options are Enabled and Disabled.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTeamsCallingPolicy.md:364]:
```suggestion
Setting this parameter allows you to control the tenant users' ability to launch an external website URL automatically in the browser window upon incoming PSTN calls for specific users or user groups. Valid options are Enabled and Disabled.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @noramicrosoft, kindly commit the few minor suggested changes. Many thanks.

**Review by get-itips — APPROVED** [review]

**Comment by noramicrosoft** [comment]
> Copy review - @noramicrosoft, kindly commit the few minor suggested changes. Many thanks.

Thanks for the comments. Updated the PR.

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**